src/nupic/experimental/ExtendedTemporalMemory.cpp

```diff
@@ -1300,6 +1331,11 @@ void ExtendedTemporalMemory::write(ExtendedTemporalMemoryProto::Builder& proto)
     matchingApicalSegmentOverlaps[i].setOverlap(
       matchingApicalSegments_[i].overlap);
   }
+
+  NTA_CHECK(learnOnOneCell_ == false) <<
+    "Serialization is not supported for learnOnOneCell";
```

Comment:
Good catch, I meant to do this in `save`.

Serialization isn't supported at all in the Python ETM. And we're not really using learnOnOneCell, I just implemented it so that I didn't have to keep disclaiming that the C++ ETM was missing learnOnOneCell.